pkg/scorecard_result.go

```diff
@@ -86,9 +87,27 @@ func (r *ScorecardResult) AsCSV(showDetails bool, writer io.Writer) error {
 }
 
 func (r *ScorecardResult) AsString(showDetails bool, writer io.Writer) error {
-	fmt.Fprintf(writer, "Repo: %s\n", r.Repo)
+	type result struct {
```

Comment:
Rather than mostly duplicate an existing struct, could we do something like:

```go
        sortedChecks := make([]checker.CheckResult, len(r.Checks))
        copy(sortedChecks, r.Checks)
        sort.Slice(sortedChecks, func(i, j int) bool {
                if sortedChecks[i].Pass == sortedChecks[j].Pass {
                        return sortedChecks[i].Name < sortedChecks[j].Name
                }
                return sortedChecks[i].Pass
        })
```

Reply:
Thanks, I agree. Fixed it 👍